ZocDoc.Bundler.nuspec

```diff
@@ -2,7 +2,7 @@
 <package xmlns="http://schemas.microsoft.com/packaging/2010/07/nuspec.xsd">
   <metadata>
     <id>ZocDoc-Bundler</id>
-    <version>1.0.2</version>
+    <version>1.0.3</version>

```

Comment:
Bumping the version number so I can push this to nuget and run a CIOD with this new Bundler code.